src/coreclr/src/vm/eventtrace.cpp

```diff
@@ -3152,6 +3153,69 @@ CrstBase * ETW::TypeSystemLog::GetHashCrst()
     return &AllLoggedTypes::s_cs;
 }
 
+// The number of type load operations
+// NOTE: This isn't the count of types loaded, as some types may have multiple type loads
+//       occur to them as they transition up type loader levels
+UINT32 s_TypeLoadOps = 0;
+
+UINT32 ETW::TypeSystemLog::TypeLoadBegin()
+{
+    CONTRACTL{
+        NOTHROW;
+        GC_TRIGGERS;
+    } CONTRACTL_END;
+
+    UINT32 typeLoad = InterlockedIncrement(&s_TypeLoadOps);
```

Comment:
Is there any value to incrementing this counter when we aren't tracking the events? May as well move the Increment inside the if body.

Reply:
This is something that I've wanted for a while as a performance counter like thing. I don't have good evidence as to whether or not its really all that useful, but this should be enough to see the value. Its sufficiently cheap that I would just like to enable it unconditionally.

Reply:
happy to keep it as long as you find it useful : )